tools/cli/services/create.js

```diff
@@ -1,25 +1,70 @@
-const fs = require('fs');
-const execute = require('../utils/execute');
-const { red } = require('../utils/colors');
-const { join } = require('path');
+const fs = require("fs");
+const execute = require("../utils/execute");
+const { red } = require("../utils/colors");
+const { join } = require("path");
+
+function runNpmInstallInDirectoryAndExitIfFailed(directoryName) {
+  try {
+    execute(
+      `npm install --prefix ${directoryName}`,
```

Comment:
Don't use the prefix argument. Prefer to pass cwd to the options object (third argument of the execute function).